src/stories/RadioButton.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import React, {ChangeEvent, useState} from 'react'
+import {Meta} from '@storybook/react'
+import styled from 'styled-components'
+
+import {BaseStyles, Box, RadioButton, RadioButtonProps, Text, ThemeProvider} from '..'
+import {COMMON, get} from '../constants'
+
+export default {
+  title: 'Forms/Radio Button',
+  component: RadioButton,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Box sx={{pt: 3}}>{Story()}</Box>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ],
+  argTypes: {
+    sx: {
+      table: {
+        disable: true
+      }
+    },
+    disabled: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    checked: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    }
+  }
+} as Meta
+
+const StyledLabel = styled.label`
+  user-select: none;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 18px;
+  margin-left: 8px;
+  display: flex;
+  cursor: pointer;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &[aria-disabled='true'] {
+    pointer-events: none;
+    cursor: not-allowed;
+    color: ${get('colors.primer.fg.disabled')};
+  }
+  ${COMMON}
```

Comment:
we don't need to pass `COMMON` props anymore.

Reply:
Cheers @pksjce . Forgot to remove these from the stories.. that's done now 👍 . Also removed from Checkbox one, as it was there too.